docs/designers-developers/developers/data/data-core.md

```diff
@@ -71,6 +71,19 @@ _Returns_
 
 -   `?Array`: An array of autosaves for the post, or undefined if there is none.
 
+<a name="getCurrentUndoOffset" href="#getCurrentUndoOffset">#</a> **getCurrentUndoOffset**
+
+Returns the current undo offset for the
```

Comment:
What does "offset" mean in that context?

Reply:
I'll clarify.